site/public/js/resizable-panels.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Initializes Resizables two panels mode
+ * where width from one panel can be given out to another one i.e resizable widths
+ * @param leftPanelSel DOM selector for left panel
+ * @param dragBarSel DOM selector for drag-bar on which this resizing event will be attached
+ */
+function initializeResizablePanels (leftPanelSel, dragBarSel) {
+  // Select all the DOM elements for dragging in two-panel-mode
+  const leftPanel = document.querySelector(leftPanelSel);
+  const panelCont = leftPanel.parentElement;
+  const dragbar = document.querySelector(dragBarSel);
+
+  let xPos = 0, yPos = 0, leftPanelWidth = 0;
+
+  // Width of left side
+  const mouseDownHandler = function(e) {
```

Comment:
Can you make this function consistent with the rest, either use all arrow functions or all anonymous functions

Reply:
Done